docs/config.md

```diff
@@ -17,21 +14,20 @@ from starlette.datastructures import CommaSeparatedStrings, Secret
 config = Config(".env")
 
 DEBUG = config('DEBUG', cast=bool, default=False)
-DATABASE_URL = config('DATABASE_URL', cast=databases.DatabaseURL)
+DATABASE_URL = config('DATABASE_URL')
```

Comment:
I'm a bit skeptical about this, because if the user prints or logs it, it will show the password, and that's really not something we want to encourage...

Reply:
Well they shouldn't log it or print it, we can't control what user will do. It's only about the database, what if they log or print passwords, sensitive information, secrets, etc?

Reply:
Can we copy `DatabaseURL` to `Starlette`? 🤔

Reply:
Actually, what's the problem of point out to `databases.DatabaseURL` here? We are not using any database feature there 🤔

Reply:
Ok, I'll the link to the databases code.

Reply:
Sorry, I didn't express myself very well on the last message. I mean, why not using the `databases.DatabaseURL` import here?

Reply:
Let's just link

Reply:
Ok, I'll do that, BTW I think I already mentioned we could be using `Secret` for the Database URL too, it will avoid the leak in the logs and also doesn't need any extra code or library. Just a thought.

Reply:
But the only thing you want to omit is the password field, the rest you actually want to see.

Reply:
yeah that's true.

Reply:
Done, please check the wording.